### PR TITLE
fix font error

### PR DESCRIPTION
**Pull request by bsima**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ set the `GHC_VERSION` environment variable:
 GHC_VERSION=ghc881 nix-build
 ```
 
-To find out which GHC versions your nix setup provides:
+To find out which GHC versions our nix setup provides:
 
 ```
-$ nix eval '(builtins.attrNames ((import <nixpkgs> {}).haskell.compiler))'
-[ "ghc822Binary" "ghc844" "ghc863Binary" "ghc865" "ghc881" "ghcHEAD" "ghcjs" "ghcjs86" "integer-simple" ]
+$ nix eval '(builtins.attrNames (import (builtins.fetchTarball (import ./nix/nixpkgs.nix)) {}).haskell.compiler)'
```
Comment (turion) [inline CONTRIBUTING.md:22]: I just tried that, it doesn't work. Did you mean to commit the file `./nix/nixpkgs.nix`?

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,10 @@
   "language": "nix",
   "env": {
     "jobs":
-      [ "GHC_VERSION=ghc844"
-      , "GHC_VERSION=ghc865"
-      , "GHC_VERSION=ghc883"
+      [ "GHC_VERSION=ghc8104"
+      , "GHC_VERSION=ghc884"
+      , "GHC_VERSION=ghc901"
+      , "GHC_VERSION=ghcHEAD"
```
Comment (turion) [inline .travis.yml:8]: Thanks for bumping! Do the older versions (8.6 and 8.4) not work anymore with this change? Or can we keep them?

**Review by turion — CHANGES_REQUESTED** [review]
Code looking great, some questions about the nix setup

File: CONTRIBUTING.md (hunk 1 above)
Comment (bsima) [reply CONTRIBUTING.md:22]: Hm, I added this file in https://github.com/sebastiaanvisser/clay/pull/213/files#diff-7b06b62d006ecf7b5a6cd3623976d7a6f1522bb1208ae904f0501347faee6b5d

Did you pull the latest?

File: .travis.yml (hunk 2 above)
Comment (bsima) [reply .travis.yml:8]: The older versions don't exist in the pinned nixpkgs version. We can add them if necessary...  but I'd have to do some research/investigation for how to support that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (turion) [reply CONTRIBUTING.md:22]: Ah, my bad. I was on an outdated branch. Yes, this works fine :)

**Comment by turion** [comment]
Thanks!

**Pull request merged**